query-connector/src/app/constants.ts

```diff
@@ -319,6 +319,7 @@ export interface ValueSet {
   valueSetId: string;
   valueSetVersion: string;
   valueSetName: string;
+  valueSetExternalId?: string;
```

Comment:
I thought we were punting on this schema change until Marcelle and I were caught up with existing DB changes and migrations?

Reply:
Ahh, apologies for not getting everyone on the same page. 

The conclusion I came away with was
- We agreed that it's helpful / not too disruptive to include the VSAC OID in schema for use elsewhere. Am reasoning that since we're adding a new property, things won't be too disruptive, but let me know if you feel differently.
- Instead of naming it `valueSetOid`, we can name it `valueSetExternalId` since it generically covers future non-VSAC ID cases. This is mostly a cosmetic change since the underlying ID itself is always going to be OID now until we support a non-VSAC use case. That is, we're not changing anything about the underlying schema other than including a newly named field that used to be there.
- Since we don't know the future shape / conventions of ID's from other terminologies, we're going to punt on how we replace the actual values that we use to construct the ID's. That is, we might change the way we construct `valueSetId`'s in the future to potentially not use this `externalId` value. This is a non-cosmetic change to how external ID's interact with the primary keys for the table. 

Hopefully this clarifies things, but if not, happy to hop on a call to explain further. Happy also just to change this to `valueSetOid` if that makes things more straightforawrd: not too attached to the naming here as long as we can have the field broken out for use elsewhere.

Reply:
Ahhh that third bullet point helps clarify things a lot. Thanks for the write up here, Bob. This definitely clarifies the intent and I agree as it stands this isn't particularly disruptive or blocking. Just wanted to get back on the same page, but I'm there now. Thanks!